dcm4che-net-audit/src/main/java/org/dcm4che3/net/audit/SyslogProtocolHandler.java

```diff
@@ -103,7 +104,7 @@ private static int getThreadPoolCoreSize() {
                 System.getProperty(AUDIT_THREAD_POOL_CORE_SIZE, String.valueOf(MINIMUM_NUMBER_OF_THREADS)));
         threadPoolSize = Math.max(Runtime.getRuntime()
                 .availableProcessors(), threadPoolSize);
-        // LOG.warn("Using audit thread pool core size {}", threadPoolSize);
+        SyslogLogger.LOG.debug("Audit core thread pool size: {}", threadPoolSize);
```

Comment:
Just use LOG here?

Reply:
Believe me, tried that but it results in an exception thrown, perhaps because of something _special_ about how the class is instantiated. While ugly, creating this private logging class worked.